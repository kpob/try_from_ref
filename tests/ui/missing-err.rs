#![no_main]

#[derive(derive_try_from::TryFromRef)]
#[source(SourceStruct)]
struct MyStruct {
    field: u32
}