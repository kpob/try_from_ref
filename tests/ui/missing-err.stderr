error: Missing err attribute

         = help: Add #[err(Type)] attribute

 --> tests/ui/missing-err.rs:5:8
  |
5 | struct MyStruct {
  |        ^^^^^^^^
